### PR TITLE
Update widget for no connection state

### PR DESCRIPTION
**Pull request by deeppandya**
Resolves https://github.com/brave/brave-browser/issues/12733 
Resolves https://github.com/brave/brave-browser/issues/12736 
Resolves https://github.com/brave/brave-browser/issues/12735 
Resolves https://github.com/brave/brave-browser/issues/12720 
Resolves https://github.com/brave/brave-browser/issues/12757

## Submitter Checklist:

- [x] There is a [ticket](https://github.com/brave/brave-browser/issues) for my issue.
- [ ] Used Github [auto-closing keywords](https://help.github.com/articles/closing-issues-via-commit-messages/) in the commit message.
- [ ] Wrote a good [PR/commit description](https://google.github.io/eng-practices/review/developer/cl-descriptions.html)
- [ ] Added appropriate labels (`QA/Yes` or `QA/No`; `release-notes/include` or `release-notes/exclude`; `OS/...`) to the associated issue
- [ ] Checked the PR locally: `npm run test -- brave_browser_tests`, `npm run test -- brave_unit_tests`, `npm run lint`, `npm run gn_check`, `npm run tslint`
- [ ] Ran `git rebase master` (if needed).
- [ ] Requested a security/privacy review as needed.

## Reviewer Checklist:

- [ ] New files have MPL-2.0 license header.
- [ ] Adequate test coverage exists to prevent regressions
- [ ] Major classes, functions and non-trivial code blocks are well-commented
- [ ] Changes in component dependencies are properly reflected in `gn`
- [ ] Code follows the [style guide](https://chromium.googlesource.com/chromium/src/+/HEAD/styleguide/c++/c++.md)
- [ ] Test plan is specified in PR before merging

## After-merge Checklist:

- [ ] The associated issue milestone is set to the smallest version that the
  changes has landed on.
- [ ] All relevant documentation has been updated, for instance:
  - [ ] https://github.com/brave/brave-browser/wiki/Deviations-from-Chromium-(features-we-disable-or-remove)
  - [ ] https://github.com/brave/brave-browser/wiki/Proxy-redirected-URLs
  - [ ] https://github.com/brave/brave-browser/wiki/Fingerprinting-Protections
  - [ ] https://github.com/brave/brave-browser/wiki/Brave%E2%80%99s-Use-of-Referral-Codes
  - [ ] https://github.com/brave/brave-browser/wiki/Custom-Headers
  - [ ] https://github.com/brave/brave-browser/wiki/Web-Compatibility-Exceptions-in-Brave
  - [ ] https://github.com/brave/brave-browser/wiki/QA-Guide
  - [ ] https://github.com/brave/brave-browser/wiki/P3A

### CODE REVIEW

File: browser/ui/android/strings/android_brave_strings.grd
Comment (srirambv) [reply browser/ui/android/strings/android_brave_strings.grd:1538]: cc: @jamesmudgett

**Review by SergeyZhukovsky — APPROVED** [review]
++ from side, just the string I mentioned needs to be checked.

**Comment by deeppandya** [comment]
> ++ from side, just the string I mentioned needs to be checked.

thank @SergeyZhukovsky i will wait on the string before i merge.

**Comment by srirambv** [comment]
Verification passed on OnePlus 6T with Android 10 running 1.19.19.x64 nightly build
Verified brave/brave-browser#12733
- Verified counter doesn't stop at a specific value when the network is disconnected. Instead shows a message that preview conversion failed


Verified brave/brave-browser#12736
- Verified no race condition while converting asset after disconnecting and reconnecting from network

Verified brave/brave-browser#12735
- Verified when there is no network widget is not disconnected instead an error message is shown when tapped on the widget or if the bottom sheet is opened

 |  
-- | --

Verified brave/brave-browser#12720
- Verified text is not cut off when the device is set to a non-EN locale

  | 
-- | --

Verified brave/brave-browser#12757
- Verified account balance is not cached and shown when widget is removed and added back via edit stack

**Pull request merged**
